cornell-dti/idol
# Implement Footer

### Summary 

Implement footer 

### Figma File 

[Nav bar and Footer](https://www.figma.com/file/6hSXHwtXblzSaR6rFV2b8p/SUMMER22-Website-Dev-Handoff?node-id=5%3A224)


### Test Plan 

- Check that implementation is consistent with design
- Check that links link to the correct URL 
### Notes 
**Unrelated changes**
- Added `dti-website-redesign` as a yarn workspace
- Added `/// <reference types="next/types/global" />` to `dti-website-redesign/next-env.d.ts` to get rid of errors for `.module.css` imports
